internal/core/src/storage/ThreadPools.cpp

```diff
@@ -53,4 +53,22 @@ ThreadPools::GetThreadPool(milvus::ThreadPoolPriority priority) {
     }
 }
 
+void
+ThreadPools::ResizeThreadPool(milvus::ThreadPoolPriority priority,
+                              int64_t size) {
+    if (size < 1) {
+        LOG_ERROR("Failed to resize threadPool, size:{}", size);
+        return;
+    }
+    std::unique_lock<std::shared_mutex> lock(mutex_);
+    auto iter = thread_pool_map.find(priority);
+    if (iter == thread_pool_map.end()) {
+        LOG_ERROR("Failed to find threadPool, priority:{}", priority);
+        return;
+    }
+    iter->second->min_threads_size_ = size;
+    iter->second->max_threads_size_ = size;
```

Comment:
I remeber folly has something like `setNumThreads`. Directly setting fields looks weird to me